Jenkinsfile

```diff
@@ -24,11 +26,27 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
             coreos-assembler fetch
             """)
         }
+
+        def prevBuildID = null
+        if (utils.shwrap_rc("test -f builds/latest")) {
+            prevBuildID = utils.shwrap_capture("readlink builds/latest")
+        }
+
         stage('Build') {
             utils.shwrap("""
             coreos-assembler build
             """)
         }
+
+        def newBuildID = utils.shwrap_capture("readlink builds/latest")
+        if (prevBuildID == newBuildID) {
+            currentBuild.result = 'SUCCESS'
+            currentBuild.description = "💤 (no new build)"
+            return
+        } else {
+            currentBuild.description = "⚡ ${newBuildID}"
```

Comment:
i'm guessing this shows up in the jenkins web interface somehow?

Reply:
Yup, exactly! It's in the build description text that appear in the build history on the left margin of the page.